extra/modules/greenbids-real-time-data/src/main/java/org/prebid/server/hooks/modules/greenbids/real/time/data/core/ThrottlingThresholds.java

```diff
@@ -0,0 +1,22 @@
+package org.prebid.server.hooks.modules.greenbids.real.time.data.core;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+import java.util.List;
+
+@Value(staticConstructor = "of")
+public class ThrottlingThresholds {
+
+    @JsonProperty("thresholds")
+    List<Double> thresholds;
+
+    @JsonProperty("tpr")
+    List<Double> tpr;
+
+    @JsonProperty("fpr")
+    List<Double> fpr;
+
+    @JsonProperty("version")
+    String version;
```

Comment:
these two are never used

Reply:
Removed fields from class definition. As they are still output by our training server and present in json have added `@JsonIgnoreProperties(ignoreUnknown = true)` annotation to ignore unused fields